src/granite/settings.cr

```diff
@@ -2,16 +2,15 @@ require "logger"
 
 module Granite
   class Settings
-    property database_url : String? = nil
-    property logger : Logger
+    class_getter adapters = [] of Granite::Adapter::Base
+    class_property logger = Logger.new STDOUT
```

Comment:
I wonder if we should change this to `::Logger.new(nil)`.  This potentially could be a bottleneck if all logging is sent to a single IO class instance.  @robacarp wdyt?

Reply:
Does having it at the class level matter?  Before, `@logger` was defaulted to `Logger.new STDOUT` as well.  Was just keeping that behavior.

Reply:
In practice this will almost always be initialized to STDOUT, or a logger pointed at STDOUT. But I think the default should be nil. It's frustrating when including a library suddenly generates a bunch of output in a program and requires additional configuration to silence it. For example, imagine including Granite in a console application which reads sqlite files -- Granite would pollute the user interface by default. The logger facility is a great tool for use of granite, but silence feels like a nice default.